tests/resources/VCH-Util.robot

```diff
@@ -76,10 +76,16 @@ Set Test Environment Variables
     Set Test VCH Name
     # cleanup any potential old certs directories
     Remove Directory  %{VCH-NAME}  recursive=${true}
+    Wait Until Keyword Succeeds  5x  1s  Create Unique Bridge Network
```

Comment:
I'd give this a longer timeout. We're expecting an active failure during the collision path so timeouts are a different matter.